src/routes/album.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const albumController = require("../controllers/album");
+
+const albumRouter = express.Router();
+
+albumRouter.post("/artists/:id/albums", albumController.createAlbum);
```

Comment:
This is the tricky route to define as the base route comes off artist router, but the controller is an album controller. There a a couple of options of how to handle this. My preferred way is to move this route into the artist.js file and import the createAlbum controller into there. Let me know if you need help with this.